traffic_portal/app/src/common/modules/form/deliveryService/form.deliveryService.DNS.tpl.html

```diff
@@ -343,6 +342,34 @@ <h3 ng-if="!open()">Previous Value</h3>
                             </div>
                         </div>
                     </div>
+                    <div class="form-group" ng-class="{'has-error': hasError(generalConfig.requiredCapabilities), 'has-feedback': hasError(generalConfig.requiredCapabilities)}">
+                        <label class="has-tooltip control-label col-md-2 col-sm-2 col-xs-12" for="capability">Required Capability
```

Comment:
labels can only be associated with one control, and this one is currently associated with every checkbox for DSRCs. I think what you really wanna do here is make a section of the form and add a section label for it, which is best accomplished by `fieldset` and `legend`, respectively.